manim/animation/creation.py

```diff
@@ -1,4 +1,40 @@
-"""Animate the display or removal of a mobject from a scene."""
+r"""Animate the display or removal of a mobject from a scene.
+
+.. manim:: CreationModule
+    :hide_source:
+
+    from manim import ManimBanner
+
+    class CreationModule(Scene):
+        def construct(self):
+            texts = [Text('manim'), Text('manim')]
+            texts[0].shift(LEFT * 2 + UP)
+            texts[1].shift(RIGHT * 2 + UP)
+            self.add(*texts)
+
+            objs = [ManimBanner().scale(0.25) for _ in range(5)]
+            for idx, obj in enumerate(objs):
+                obj.shift(LEFT * 6 + RIGHT * (2.75 * idx) + DOWN)
+            self.add(*objs)
+
+            self.play(
+                # text creation
+                Write(texts[0]),
+                AddTextLetterByLetter(texts[1]),
+
+                # mobject creation
+                ShowCreation(objs[0]),
+                Uncreate(objs[1]),
+                DrawBorderThenFill(objs[2]),
+                ShowIncreasingSubsets(objs[3]),
+                ShowSubmobjectsOneByOne(objs[4]),
+
+                run_time=3,
+            )
+
+            self.wait()
+
+"""
```

Comment:
I think this example looks a bit overcrowded and does not show the functionalities in the best way.
https://manimce--753.org.readthedocs.build/en/753/reference/manim.animation.creation.html?highlight=creation#module-manim.animation.creation

Reply:
I totally agree! This is not finished by any means. I was waiting for ManimBanner to be merged before continuing work here.

Reply:
Ahh, perfect!